main/src/ca/uwaterloo/flix/language/phase/unification/BoolAlg.scala

```diff
@@ -101,7 +101,16 @@ trait BoolAlg[F] {
     *
     * This environment should be used in the functions [[toType]] and [[fromType]].
     */
-  def getEnv(fs: List[Type]): Bimap[Symbol.KindedTypeVarSym, Int]
+  def getEnv(fs: List[Type]): Bimap[Symbol.KindedTypeVarSym, Int] = {
+    // Compute the variables in `tpe`.
+    val tvars = fs.flatMap(_.typeVars).map(_.sym).to(SortedSet)
```

Comment:
Technically this would be faster as a straight fold over fs that constructs a SortedSet.